lib/miq_expression.rb

```diff
@@ -1,7 +1,7 @@
 class MiqExpression
   require_nested :Tag
   include Vmdb::Logging
-  attr_accessor :exp, :context_type, :preprocess_options
+  attr_accessor :exp, :context_type, :preprocess_options, :virtual_custom_attributes
```

Comment:
Is there a reason to add this? The data it is encapsulating is being memoized elsewhere